src/std/misc/walist.ss

```diff
@@ -0,0 +1,91 @@
+(import
+  :std/misc/alist
+  :std/sugar)
+
+(export
+  _walist_ _walist_?
```

Comment:
can we avoid exporting those two?

Reply:
or at least rename them, these are ugly names.

Reply:
What names would you suggest? They are what std/text/json uses to recognize it's being given a walist.

Reply:
base-walist ? wrapped-alist ?

Reply:
lets call the base walist, thats what the users will bw working with.
Also, the accessor should be export renamed to walist->list i think.

Reply:
If the base is walist, what is the pure equal one called? The same, and I override the rest? Then it's not final?

Reply:
Why `walist->list` rather than `walist->alist` ?